src/styles/utils/variables.scss

```diff
@@ -0,0 +1,11 @@
+$dark-blue: rgba(6, 11, 53, 1);
+$grey: rgba(97, 96, 112, 1);
+$white: #fff;
+$blue: rgba(0, 172, 220, 1);
+$title-hover: rgb(52, 86, 139);
+$columns-count: (
```

Comment:
The variable `$columns-count` is defined as a map, but it is missing the closing parenthesis for the map. Make sure to close the map properly.